operators/hack/deployer/config/plans.yml

```diff
@@ -14,3 +14,14 @@ plans:
     localSsdCount: 1
     nodeCountPerZone: 1
     gcpScopes: https://www.googleapis.com/auth/devstorage.read_only,https://www.googleapis.com/auth/logging.write,https://www.googleapis.com/auth/monitoring,https://www.googleapis.com/auth/servicecontrol,https://www.googleapis.com/auth/service.management.readonly,https://www.googleapis.com/auth/trace.append
+- id: aks-ci
+  operation: create
+  clusterName: ci
+  provider: aks
+  kubernetesVersion: 1.12.8
+  machineType: Standard_D16s_v3
```

Comment:
Feels a little bit too much. Why not `Standard_D8s_v3`? It will be closer to what we are using in GKE